client/helpers/files.ts

```diff
@@ -13,8 +13,15 @@ export interface UploadFileReq extends BaseReq {
   /** A {@link CreateFileRequest}. */
   readonly createFileReq: CreateFileRequest;
 
-  /** File data, use {@link Buffer} or {@link ReadStream} in Node. */
-  readonly fileData: unknown;
+  readonly file?: {
+    readonly data: Buffer;
+    readonly size: number;
+  };
```

Comment:
The client might be a bit easier to use if it accepted a path and handled reading the file into a buffer and determining the size.